libr/include/r_heap_glibc.h

```diff
@@ -150,6 +150,17 @@ typedef struct r_tcache_perthread_struct_64 {
 	ut64 entries[TCACHE_MAX_BINS];
 } RHeapTcache_64;
 
+typedef struct r_tcache_perthread_struct_old_32 {
```

Comment:
as said in the issue, I would give this a more descriptive name than "old/new". "pre_227_" seems better to me.

Reply:
Ok I prefer old and put a comment on the version, but I will change it to pre_230 since it's for versions <= 2.29